community/kernel/src/main/java/org/neo4j/kernel/impl/store/id/IdGeneratorImpl.java

```diff
@@ -372,6 +367,33 @@ public synchronized void close()
         }
     }
 
+    private void closeChannelAndPutThisInstanceIntoClosedState() throws IOException
```

Comment:
yes correct, it's just naming that i don't like. Method should do just one thing and `And` in the name suggesting that it's not the case